.github/workflows/daily.yml

```
@@ -19,6 +19,45 @@ on:
         default: 'master'
 
 jobs:
+  tests-with-elle:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install build dependencies
+        run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov leiningen
+      - name: Build
+        run: |
+          mkdir build && cd build
+          cmake .. -DPYTEST_OPTS="--redis-executable=redis/src/redis-server --elle-cli=elle-cli/target/elle-cli-0.1.4-standalone.jar --elle -v"
+          make
+      - name: Checkout Redis
+        uses: actions/checkout@v2
+        with:
+          repository: 'redis/redis'
+          ref: 'unstable'
+          path: 'redis'
+      - name: Build Redis
+        run: cd redis && make -j 4 gcov
```

```suggestion
        run: cd redis && make -j 4
```

I guess this is copy paste error from coverage build. Let's delete it